e2e/test/preview.test.ts

```diff
@@ -13,6 +13,22 @@ test('user can see single preview tab', async ({ page }) => {
   await expect(preview.getByText('Index page')).toBeVisible();
 });
 
+test('user can reload the page', async ({ page }) => {
+  await page.goto(`${BASE_URL}/single`);
+
+  const preview = page.frameLocator('[title="Node Server"]');
+
+  await expect(preview.getByText('Index page')).toBeVisible();
+
+  await page.getByTestId('write-to-file').click();
+
+  await expect(preview.getByText('Index page')).toBeVisible();
+
+  await page.getByTitle('Reload Preview').click();
```

Comment:
`byTitle` is good, but we can improve this. Let's also verify the element's role:

```suggestion
  await page.getByRole('button', { name: 'Reload Preview' }).click();
```

Now the fact that the name comes from `title` is just an implementation detail, and we can verify that proper element/role is used for this clickable element.